README.md

```diff
@@ -21,6 +21,9 @@ The algorithm is as follow :
 
 :construction:
 
+### Install JAVA 11
+
```

Comment:
I would have done here a bullet list with the java run time, maven and linux-based OS since RAPL is only available on Linux